docs/troubleshooting-guide/collection-permissions.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Troubleshooting collection permissions
+---
+
+# Troubleshooting collection permissions
+
+Use this guide if you're having trouble with permissions to _view or edit_ questions or dashboards.
```

Comment:
Should we flip this? Collections permissions are the issue, but would people know that it's a collection permissions issue? Or should we frame this page as troubleshooting viewing and editing questions and dashboards?